src/ch01-00-foreword.md

```diff
@@ -0,0 +1,39 @@
+# Foreword
+
+In recent years, it has become an industry standard to deploy software using
+some form of containerization; and while this trend has helped to reduce the
+complexity of our deployments by offering process and file isolation at
+runtime, it has done virtually nothing to reduce the complexity of building
+and maintaining software.
+
+Nix tries to bridge this gap using the same isolation primitizes used by
```

Comment:
```suggestion
Nix bridges this gap using the same isolation primitives used by
```

Reply:
whoops, good catch

Reply:
also, I used "tries to" here to anticipate the nay sayers who will inevitably crop up and point out that nix does not ensure reproducible binaries in absolutely every case atm. While this isn't exactly Nix's fault, is usually related to the code or compiler exhibiting not deterministic behavior, we know that people don't always lean toward absolute reason either.

Reply:
As a statement of confidence, I'd try to avoid 'hedging' claims too much too early in the foreword. Valid objections on detail seem like a layer violation of the attentive reader, here.

Reply:
You're right. And this isn't the way I would have wrote it if I was overthinking it to begin with :smile: